src/snowflake/cli/plugins/nativeapp/exceptions.py

```diff
@@ -23,16 +23,16 @@ def __init__(self, name: str):
 
 
 class ApplicationAlreadyExistsError(ClickException):
-    """An application not created by SnowCLI exists with the same name."""
+    """An application object not created by Snowflake CLI exists with the same name."""
```

Comment:
Just curious. If I create an application object via SQL, can I alter it with SnowCLI? We discussed that the idea is that customers user one or the other, but I'm curious if there are any specific prohibitions.

Reply:
Yes, a user can interact with objects created via SQL using the Snowflake CLI, assuming they provide the correct names etc to a project definition file. The Snowflake CLI will generally warn the user in that case.